### ISSUE TITLE
docusaurus-plugin-typedoc: Plugins not loading in docusaurus after updating typedoc to 0.24

### ISSUE THREAD

**Issue by CoderIllusionist** [tgreyuk/typedoc-plugin-markdown#416]
When updating `typedoc` to `0.24.x` and `docusaurus-plugin-typedoc` to `0.19.0` plugins aren't loading anymore

docusaurus.config.js: 
  ``` javascript
  plugins: [
    [
      'docusaurus-plugin-typedoc',
      {
        entryPoints: ['../lib/index.ts'],
        tsconfig: '../lib/tsconfig.json',
        out: 'api',
        plugin: ['typedoc-plugin-replace-text'],
      },
    ],
  ],
```

Before the update I could see in the cli:
`Loaded plugin typedoc-plugin-replace-text`

When updating I get the following

```
[error] Tried to set an option (replaceText) that was not declared. You may have meant:
        readme
        pretty
        readmeTitle
        emit
        name
        sort
        help
```

(worked flawlesly)
old `package.json`
```json
{
    "docusaurus-plugin-typedoc": "0.18.0"
    "typedoc": "0.23.28",
    "typedoc-plugin-markdown": "3.14.0",
    "typedoc-plugin-replace-text": "2.1.0",
}
```

(see error above)
new `package.json`
```json
{
    "docusaurus-plugin-typedoc": "0.19.0"
    "typedoc": "0.24.4",
    "typedoc-plugin-markdown": "3.15.1",
    "typedoc-plugin-replace-text": "2.2.0",
}
```

in `typedoc.json`

```
{
  "replaceText": {
    "inCodeCommentText": true,
    "inCodeCommentTags": true,
    "inIncludedFiles": true,
    "replacements": [
        // replacements here
    ]
  }
}
```

Any thoughts?

**Comment by tgreyuk** [comment]
Sorry - there was an API change in TypeDoc 0.24 that wasn't implemented. 

Should be fixed in docusaurus-plugin-typedoc@0.19.1

**Comment by CoderIllusionist** [comment]
Thanks! Fixed!
